streams.groovy

```diff
@@ -1,7 +1,9 @@
 // Canonical definition of all our streams and their type.
 
+next_devel = load("next-devel/status.groovy")
```

Comment:
This is assuming that the repo is checked out exactly at the working directory. This is true for all loads of `streams.groovy` right now, but it'd be easy to regress on. A simpler Groovy approach (but trickier implementation-wise) is to have `manage.py` edit `streams.groovy` directly, but definitely good to try this!

Reply:
All of the loads of `streams.groovy` also assume that, so I think the only risk is from external code using it as a library?

It seemed cleaner to keep the generated code separate, but I'm open to direct editing if we want/need.  Will merge as-is for now.

Reply:
Turns out the `load()` doesn't work.  Followup in https://github.com/coreos/fedora-coreos-pipeline/pull/453.